setup.cfg

```diff
@@ -34,3 +34,22 @@ ignore_missing_imports = True
 [flake8]
 max-line-length = 200
 exclude = venv/*,var/*
+
+# E203 - White space before ':', conflicting with black
+# E712 - comparison to False should be 'if cond is False:' or 'if not cond:', doesn't apply to our code
+# E731 - do not assign a lambda expression, use a def, doesn't apply to our code
+
+# F401 - module imported but not used
+
+# W503 - line break before binary operator, conflicting with black
+# W605 - invalid escape sequence, modifying will cause test failures
+
+ignore = E203,E712,E731,W503
+per-file-ignores =
+  #  JsonApiClient used in comments, black doesn't inline comments in that line
+  codalab/lib/bundle_util.py: F401,F841
+  #  Commented explicitly that we want to keep the imports.
+  codalab/server/rest_server.py: F401
```

Comment:
Can you update the comment to `# Keep the unused imports as they are used to route the rest service`?

Reply:
done